src/PowerShellEditorServices/Services/DebugAdapter/Handlers/ThreadsHandler.cs

```diff
@@ -16,13 +16,8 @@ public Task<ThreadsResponse> Handle(ThreadsArguments request, CancellationToken
         {
             return Task.FromResult(new ThreadsResponse
             {
-                // TODO: What do I do with these?
-                Threads = new Container<OmniSharp.Extensions.DebugAdapter.Protocol.Models.Thread>(
-                    new OmniSharp.Extensions.DebugAdapter.Protocol.Models.Thread
-                    {
-                        Id = 1,
-                        Name = "Main Thread"
-                    })
+                // TODO: This is an empty container of threads...do we need to make a thread?
+                Threads = new Container<System.Threading.Thread>()
```

Comment:
I thought this was weird, but we already didn't really know what to do with it (see "What do I do with these?")